src/Service/Iam/README.md

```diff
@@ -0,0 +1,20 @@
+# AsyncAws IAM Client
+
+![](https://github.com/async-aws/iam/workflows/Tests/badge.svg?branch=master)
+![](https://github.com/async-aws/iam/workflows/BC%20Check/badge.svg?branch=master)
+
+An API client for IAM.
+
+## Install
+
+```cli
+composer require async-aws/iam
+```
+
+## Documentation
+
+See https://async-aws.com/clients/iam.html for documentation.
```

Comment:
```suggestion
See https://async-aws.com for documentation.
```

Reply:
thanks. Fixed template too

Reply:
hmm.. no in fact, the issue is not here, the issue is the documentation is missing.

Reverted.

Reply:
Yeah, but documentation is not strictly required. Is it?

Reply:
I think it should. At least for SEO, and some people (me included) use the list of ApiClient menu to known the liste of implemented client.

Reply:
It is currently required to be in the list, but it is not required to have a separate page for it.